packages/i18n/locales/cs

```diff
@@ -0,0 +1,457 @@
+      "unstakedTokens": "Odemčení tokenů ${{tokenSymbol}}",
+      "unstakingMechanics": "Pokud si žetony vyberete nyní, bude trvat {{humanReadableTime}}, než vám budou k dispozici. Během této doby nebudete odměňováni žetony za umístění. Výběr žetonů nebude možné zrušit.",
+      "verifyingGovernanceToken": "Ověření tokenu CW20...",
+      "voteAgain": "Hlasovat znovu",
+      "votedOnProposal": "Hlasovali jste <0/> o návrhu.",
+      "walletConnectMobileLimitations": "Objevte uživatelské rozhraní. Vzhledem k současným omezením v systému WalletConnect nebudete moci připojit svou mobilní peněženku k blockchainu {{chain}}.",
+      "yes": "Ano",
+      "yesVote": "Ano",
+      "yesWillWinTieClarification": "'Při rovnosti hlasů zvítězí ano.",
+      "youAreMember": "Jste členem.",
+      "yourBalance": "Váš zůstatek",
+      "yourTokensUnstaked": "Vaše ${{tokenSymbol}} žetony byly vybrány."
+    },
+    "proposalModuleLabel": {
+      "cw-govmod-single": "Jednotlivý výběr",
+      "cw-proposal-single": "Jednotlivý výběr",
+      "cwd-proposal-single": "Jednotlivý výběr"
+    },
+    "splash": {
+      "anyoneCanParticipate": "Plnohodnotným účastníkem může být kdokoli z vaší komunity.",
+      "createExploreJoin": "Vytvoření, přezkoumání a členství v organizaci",
+      "cta": "Přejít do aplikace",
+      "daosCreated": "DAO bylo úspěšně založeno",
+      "documentation": "Dokumentace",
+      "easyToUse": "Snadno použitelné rozhraní",
+      "easyToUseExplanation": "Rozhraní DAO DAO zpřístupňuje DAOs všem....",
+      "joinTheCommunity": "Připojte se ke komunitě.",
+      "longTagline": "Jednoduchá, promyšlená a bezplatná správa organizace. Vytvořeno s láskou společností DAO DAO, od společnosti Juno.",
+      "poweredByJuno": "Powered by Juno",
+      "proposalsCreated": "Vytvořené návrhy ",
+      "proposeAndVote": "Návrhy a hlasy",
+      "readMore": "Zobrazit více",
+      "shortTagline": "Organizace pro všechny.",
+      "transparencyExplanation": "Každý se může podívat, jaká rozhodnutí navrhujete a kdo hlasoval pro nebo proti.",
+      "transparentGovernanceOnChain": "Transparentní správa v blockchainu",
+      "usdcTotalValue": "Hodnota USDC",
+      "whatIsADao": "Co je DAO?",
+      "whatIsADaoExplanation": "DAO je decentralizovaná autonomní organizace demokraticky řízená svými členy....."
+    },
+    "success": {
+      "addedToken": "Přidání tokenů doKeplru.",
+      "daoCreatedPleaseWait": "Organizace byla vytvořena. Na stránku budete přesměrováni, až bude připravena.....",
+      "proposalClosed": "Úspěšně uzavřeno.",
+      "proposalExecuted": "Úspěšně provedeno",
+      "voteCast": "Hlasování proběhlo úspěšně."
+    },
+    "title": {
+      "404": "404 - Stránka nebyla nalezena",
+      "500": "500 - Interní problém serveru",
+      "actions_one": "Akce",
+      "actions_other": "Akce",
+      "addTreasuryToken": "Přidání tokenu pokladny",
+      "addresses": "Adresy",
+      "allVotes": "Všechny hlasy",
+      "apr": "RPSN",
```

Comment:
RPSN je terminus technicus, znamená to úrok na půjčce včetně poplatků za správu úvěru, možná je tohle OK, jinak bych ale dal zhodnocení, neznám ten ekosystém